### DISCUSSION TITLE
Webview2 get document object

### DISCUSSION THREAD

**Discussion by frSurfer** [dotnet/winforms#4270] [Ideas] [answered]
You guys won't get any traction upon Webview2 until you give clear, easy and simple access to the html document object, like we used to do with the old webbrowser object!!!

Who is the genius that decided that to access the html document we developers must go through a full page of javascript? This is just "If we can find a way to make it more complicated, we will!".

Come on, you should be able to KISS... (Keep It Simple Sweetie)

**Comment by RussKie** [comment, marked as answer]
WebView2 has no relation to Windows Forms runtime. Please direct your feedback to the WebView2 Feedback repo: https://github.com/MicrosoftEdge/WebView2Feedback
